### DISCUSSION TITLE
Request: allow opt-out for double left click as a shortcut

### DISCUSSION THREAD

**Discussion by elad2109** [flameshot-org/flameshot#2558] [Ideas]
![image](https://user-images.githubusercontent.com/1093841/164948698-2ef77f8b-d72e-4ac5-9b14-23d755859660.png)

Often times my non-sensitive mouse makes an unplanned left double click which exits from the edit screenshot window.
Please allow to disable this shortcut.

**Comment by mmahmoudian** [comment]
Please use the issue section to file a feature request. Discussions are more for questions and discussions. I highly suggest copying this into a feature request issue and close this one.

**Reply by ding-dang-do** [reply]
#2655

**Comment by Neburski** [comment]
It would be nice to have an opt-out for double left click for the following reason:

To go back into editing mode for an object, e.g., a text field, it is necessary to double click the text. If a user accidentally miss clicks during this action, flameshot will copy the screenshot and all work has been lost, i.e., all objects disappear from the screen and are only saved in the screenshot. Thus, the user can't edit any of the objects anymore and if work was not yet completed (e.g., a review of something), the user might need to start all over again.

Btw: Flame shot is the best screenshot tool I have ever encountered. It allows me to make very quick reviews on the fly with the ability to draw, highlight, strike through, scale the canvas, ....
